src/components/ContentPlaceholder/ContentPlaceholder.scss

```diff
@@ -0,0 +1,35 @@
+@keyframes shimmer {
+    0% {
+        background-position: 100% 0;
+    }
+
+    100% {
+        background-position: -100% 0;
+    }
+}
+
+.ContentPlaceholder {
+    animation:{
+        duration: 1s;
+        fill-mode: forwards;
+        iteration-count: infinite;
+        name: shimmer;
+        timing-function: linear;
+    }
+    line-height: inherit;
+    height: auto;
+    width: 100%;
+    background: var(--color-neutral-25);
+    background: linear-gradient(to right, var(--color-neutral-10) 5%, var(--color-neutral-25) 15%, var(--color-neutral-10) 30%);
+    background-size: 200% 100%;
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+
+    &__mask {
+        background-color: white;
+        height: inherit;
+        width: 0;
```

Comment:
since it is set in the style element, it is redundant here. can it be removed? Or you think it is makes it more readable?

Reply:
See previous comments